modules/administration-guide/partials/con_che-server-with-devworkspace.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// {prod-id-short}-workspace-controller-with-devworkspace
+
+
+[id="{prod-id-short}-server-with-devworkspace_{context}"]
+= {prod-short} server
+
+The {prod-short} server is the central service of {prod-short} server-side components. 
+It simplifies the usage of the {devworkspace} API in a multitenant context.
+It is a Java web service exposing an HTTP REST API. 
+The {prod-short} deployment starts it in the `{prod-deployment}` Deployment. 
```

Comment:
I'm not sure what this last sentence means. Che deployment starts Che Deployment? 

I think this entire sentence can be omitted.